### PR TITLE
Duplicates pose2 point2 factors for parametric,

### PR DESCRIPTION
**Pull request by Affie**
TODO: consolidate with wrapper: https://github.com/JuliaRobotics/IncrementalInference.jl/issues/467

### CODE REVIEW

**Comment by andreasnoack** [comment]
# [Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=h1) Report
> Merging [#239](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=desc) into [master](https://codecov.io/gh/JuliaRobotics/RoME.jl/commit/e469f92212f3d9c0eb7435ead9d3eddd11e3f971&el=desc) will **decrease** coverage by `0.41%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #239      +/-   ##
==========================================
- Coverage   23.54%   23.12%   -0.42%     
==========================================
  Files          36       36              
  Lines        1661     1691      +30     
==========================================
  Hits          391      391              
- Misses       1270     1300      +30     
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=footer). Last update [e469f92...0ef3594](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by andreasnoack** [comment]
# [Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=h1) Report
> Merging [#239](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=desc) into [master](https://codecov.io/gh/JuliaRobotics/RoME.jl/commit/e469f92212f3d9c0eb7435ead9d3eddd11e3f971&el=desc) will **decrease** coverage by `0.41%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #239      +/-   ##
==========================================
- Coverage   23.54%   23.12%   -0.42%     
==========================================
  Files          36       36              
  Lines        1661     1691      +30     
==========================================
  Hits          391      391              
- Misses       1270     1300      +30     
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=footer). Last update [e469f92...0ef3594](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by andreasnoack** [comment]
# [Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=h1) Report
> Merging [#239](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=desc) into [master](https://codecov.io/gh/JuliaRobotics/RoME.jl/commit/e469f92212f3d9c0eb7435ead9d3eddd11e3f971&el=desc) will **decrease** coverage by `0.41%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #239      +/-   ##
==========================================
- Coverage   23.54%   23.12%   -0.42%     
==========================================
  Files          36       36              
  Lines        1661     1691      +30     
==========================================
  Hits          391      391              
- Misses       1270     1300      +30     
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=footer). Last update [e469f92...0ef3594](https://codecov.io/gh/JuliaRobotics/RoME.jl/pull/239?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: src/factors/BearingRange2D.jl [hunk 1]
```diff
@@ -44,6 +44,34 @@ function (pp2br::Pose2Point2BearingRange)(res::Array{Float64},
   return res[1]
 end
 
+#TODO wrapper
+function (s::Pose2Point2BearingRange{<:Normal})(xi::Vector{T}, lm::Vector{T}; kwargs...) where T <: Real
```
Comment (dehann) [inline src/factors/BearingRange2D.jl]: is this not going to conflict with the existing function?

Comment (dehann) [reply src/factors/BearingRange2D.jl]: https://juliarobotics.org/Caesar.jl/latest/concepts/adding_variables_factors/

**Pull request merged**